[2024-04-17T17:45:50.282+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: files_ext.create_directory manual__2024-04-17T17:45:46.448688+00:00 [queued]>
[2024-04-17T17:45:50.291+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: files_ext.create_directory manual__2024-04-17T17:45:46.448688+00:00 [queued]>
[2024-04-17T17:45:50.291+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2024-04-17T17:45:50.304+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): create_directory> on 2024-04-17 17:45:46.448688+00:00
[2024-04-17T17:45:50.311+0000] {standard_task_runner.py:60} INFO - Started process 1946 to run task
[2024-04-17T17:45:50.314+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'files_ext', 'create_directory', 'manual__2024-04-17T17:45:46.448688+00:00', '--job-id', '2156', '--raw', '--subdir', 'DAGS_FOLDER/files_extraction.py', '--cfg-path', '/tmp/tmp795llksm']
[2024-04-17T17:45:50.316+0000] {standard_task_runner.py:88} INFO - Job 2156: Subtask create_directory
[2024-04-17T17:45:50.354+0000] {task_command.py:423} INFO - Running <TaskInstance: files_ext.create_directory manual__2024-04-17T17:45:46.448688+00:00 [running]> on host fdd3cfdf3f53
[2024-04-17T17:45:50.437+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='files_ext' AIRFLOW_CTX_TASK_ID='create_directory' AIRFLOW_CTX_EXECUTION_DATE='2024-04-17T17:45:46.448688+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-17T17:45:46.448688+00:00'
[2024-04-17T17:45:50.440+0000] {logging_mixin.py:188} INFO - Directory already exists.
[2024-04-17T17:45:50.441+0000] {python.py:202} INFO - Done. Returned value was: ['dags', 'logs', 'data', 'destination', 'plugins', 'config', 'webserver_config.py', 'extraction_done', '***.cfg']
[2024-04-17T17:45:50.462+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=files_ext, task_id=create_directory, execution_date=20240417T174546, start_date=20240417T174550, end_date=20240417T174550
[2024-04-17T17:45:50.486+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-17T17:45:50.545+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
